sale_promotion_rule_display_discount_amount/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sale Promotion Rule Display Discount Amount",
+    "summary": """
+        This addon allows you to display the amount of the discount applied by
+        the sale promotion rules on the sale order""",
+    "version": "10.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/sale-workflow/",
+    "depends": ["sale_promotion_rule", "sale_discount_display_amount"],
+    "auto_isntall": "True",
```

Comment:
Fix this

Reply:
@rousseldenis Fix what?

Reply:
'auto_install' and not 'auto_isntall' :smile:

Reply:
good catch @rousseldenis !